packages/rancher-logging/generated-changes/patch/README.md.patch

```diff
@@ -0,0 +1,10 @@
+--- charts-original/README.md
++++ charts/README.md
+@@ -67,6 +67,7 @@
+ | `securityContext`                                   | Container SecurityContext for Logging operator. [More info](https://kubernetes.io/docs/concepts/policy/security-context/) | `{"allowPrivilegeEscalation": false, "readOnlyRootFilesystem": true}` |
+ | `createCustomResource`                              | Create CRDs. | `true` |
+ | `monitoring.serviceMonitor.enabled`                 | Create Prometheus Operator servicemonitor. | `false` |
++| `global.seLinux.enabled`                            | Add seLinuxOptions to Logging resources | `false` |
```

Comment:
We should add that it requires the RPM. Could also link out to docs once they are ready.

Reply:
Added a link to the rke2-selinux github release page

Reply:
That works for now, we should try to remember to swap once it moves, ideally to the official docs.